### DISCUSSION TITLE
mobile app doesn't show newly added reactions

### DISCUSSION THREAD

**Discussion by 3m4r5** [community/community#18232] [Mobile] [answered]
[iOS](https://user-images.githubusercontent.com/63432932/173014833-7e3a1974-fc97-49a0-ac37-375ad7099421.png) vs [Web](https://user-images.githubusercontent.com/63432932/173014947-9d673786-83c7-4db6-8dbc-3b7e2932be47.png)

**Comment by brianlovin** [comment, marked as answer]
@3m4r5 could you share a few more details about this — for example, if you pull to refresh, do the reactions appear?

**Reply by 3m4r5** [reply]
I tried refreshing the page & clearing cache, both didn't work.

**Reply by brianlovin** [reply]
Are you able to share a public link to this thread so that we can continue investigating on our side?

**Reply by 3m4r5** [reply]
@brianlovin yes, here's the link to the thread shown in the screenshots: https://github.com/github-community/community/discussions/18128#discussioncomment-2916074

**Reply by brianlovin** [reply]
Perfect, I've filed an issue with our team to investigate — thanks!

**Reply by 3m4r5** [reply]
no problem
